From 1f403ad229f0e2782d6427a79f0fbeb6b76148b6 Mon Sep 17 00:00:00 2001
From: Guillaume Ayoub <guillaume.ayoub@kozea.fr>
Date: Fri, 28 Sep 2018 16:13:08 +0200
Subject: [PATCH] Fix backslashes in docstring

Fix #205.
---
 cairosvg/url.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/cairosvg/url.py b/cairosvg/url.py
index 9dd9cad..1b3294d 100644
--- a/cairosvg/url.py
+++ b/cairosvg/url.py
@@ -37,8 +37,8 @@ def normalize_url(url):
 
     The input ``url`` may look like the following:
 
-        - C:\Directory\zzz.svg
-        - file://C:\Directory\zzz.svg
+        - C:\\Directory\\zzz.svg
+        - file://C:\\Directory\\zzz.svg
         - zzz.svg
 
     The output ``url`` on NT systems would look like below:
